src/components/lyne-button/lyne-button.tsx

```diff
@@ -64,27 +64,12 @@ export class LyneButton {
    */
   @Prop() public ariaHaspopup?: InterfaceButtonAttributes['popup'];
 
-  @Element() private _element: HTMLElement;
-
-  private _buttonClick = (): void => {
-    let eventDetail;
-
-    if (this.visualButtonOnly) {
-      return;
-    }
-
-    if (this.eventId) {
-      eventDetail = this.eventId;
-    }
-
-    const event = new CustomEvent(events.click, {
-      bubbles: true,
-      composed: true,
-      detail: eventDetail
-    });
-
-    this._element.dispatchEvent(event);
-  };
+  /** Emits whenever the native button click event triggers. */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'lyne-button_click'
+  }) public click: EventEmitter<any>;
```

Comment:
Can we use another type than any?

Reply:
Added todo, as it's not yet clear to me what the best type should be.